### PR TITLE
media: Add doxygen of media focus

### PR DESCRIPTION
**Pull request by junmin-kim**
Add doxygen comment in FocusChangeListener, FocusManager, FocusRequest

### CODE REVIEW

**Comment by junmin-kim** [comment]
@chanijjani Please review this PR

**Comment by seinfra** [comment]
Target : [aee277ee500471c87d37bad409dde595574861df] - Code Rule Check (C++) OK.

**Comment by seinfra** [comment]
Target : [aee277ee500471c87d37bad409dde595574861df] - Code Rule Check Result:
framework/include/media/FocusManager.h:67: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:VxO)
framework/include/media/FocusManager.h:67: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:OxV)
framework/include/media/FocusManager.h:67: ERROR: [SPC_M_OPR] spaces required around that '<' (ctx:VxV)
framework/include/media/FocusManager.h:67: ERROR: [SPC_M_OPR] spaces required around that '>' (ctx:VxW)
framework/include/media/FocusManager.h:75: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:VxO)
framework/include/media/FocusManager.h:75: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:OxV)
framework/include/media/FocusManager.h:75: ERROR: [SPC_M_OPR] spaces required around that '<' (ctx:VxV)
framework/include/media/FocusManager.h:75: ERROR: [SPC_M_OPR] spaces required around that '>' (ctx:VxW)
framework/include/media/FocusManager.h:81: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:VxO)
framework/include/media/FocusManager.h:81: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:OxV)
framework/include/media/FocusManager.h:81: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:VxO)
framework/include/media/FocusManager.h:81: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:OxV)
framework/include/media/FocusManager.h:81: ERROR: [SPC_M_OPR] spaces required around that '<' (ctx:VxV)
framework/include/media/FocusManager.h:81: ERROR: [SPC_M_OPR] spaces required around that '>' (ctx:VxW)
framework/include/media/FocusManager.h:82: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:VxO)
framework/include/media/FocusManager.h:82: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:OxV)
framework/include/media/FocusManager.h:86: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:OxV)
framework/include/media/FocusManager.h:87: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:OxV)
framework/include/media/FocusManager.h:87: ERROR: [SPC_M_OPR] spaces required around that '<' (ctx:VxV)
framework/include/media/FocusManager.h:87: ERROR: [SPC_M_OPR] spaces required around that '>' (ctx:VxW)
framework/include/media/FocusManager.h:92: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:VxO)
framework/include/media/FocusManager.h:92: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:OxV)
framework/include/media/FocusManager.h:93: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:OxV)
framework/include/media/FocusManager.h:93: ERROR: [SPC_M_OPR] spaces required around that '<' (ctx:VxV)
framework/include/media/FocusManager.h:93: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:VxO)
framework/include/media/FocusManager.h:93: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:OxV)
framework/include/media/FocusManager.h:93: ERROR: [SPC_M_OPR] spaces required around that '<' (ctx:VxV)
framework/include/media/FocusManager.h:93: ERROR: [SPC_M_OPR] need consistent spacing around '>>' (ctx:VxW)
framework/include/media/FocusManager.h:94: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:OxV)
framework/include/media/FocusRequest.h:65: ERROR: [IDT_M_TAB] code indent should use tabs where possible
framework/include/media/FocusRequest.h:65: ERROR: [IDT_M_TAB] please, no space before tabs
framework/include/media/FocusRequest.h:68: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:VxO)
framework/include/media/FocusRequest.h:68: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:OxV)
framework/include/media/FocusRequest.h:68: ERROR: [SPC_M_OPR] spaces required around that '<' (ctx:VxV)
framework/include/media/FocusRequest.h:68: ERROR: [SPC_M_OPR] spaces required around that '>' (ctx:VxW)
framework/include/media/FocusRequest.h:72: ERROR: [IDT_M_TAB] code indent should use tabs where possible
framework/include/media/FocusRequest.h:72: ERROR: [IDT_M_TAB] please, no space before tabs
framework/include/media/FocusRequest.h:75: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:OxV)
framework/include/media/FocusRequest.h:75: ERROR: [SPC_M_OPR] spaces required around that '<' (ctx:VxV)
framework/include/media/FocusRequest.h:75: ERROR: [SPC_M_OPR] spaces required around that '>' (ctx:VxW)
framework/include/media/FocusRequest.h:78: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:OxV)
framework/include/media/FocusRequest.h:79: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:OxV)
framework/include/media/FocusRequest.h:79: ERROR: [SPC_M_OPR] spaces required around that '<' (ctx:VxV)
framework/include/media/FocusRequest.h:79: ERROR: [SPC_M_OPR] spaces required around that '>' (ctx:VxW)
framework/include/media/FocusRequest.h:88: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:OxV)
framework/include/media/FocusRequest.h:95: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:OxV)
framework/include/media/FocusRequest.h:95: ERROR: [SPC_M_OPR] spaces required around that '<' (ctx:VxV)
framework/include/media/FocusRequest.h:95: ERROR: [SPC_M_OPR] spaces required around that '>' (ctx:VxW)
framework/include/media/FocusRequest.h:98: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:OxV)
framework/include/media/FocusRequest.h:99: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:OxV)
framework/include/media/FocusRequest.h:99: ERROR: [SPC_M_OPR] spaces required around that '<' (ctx:VxV)
framework/include/media/FocusRequest.h:99: ERROR: [SPC_M_OPR] spaces required around that '>' (ctx:VxW)

File: framework/include/media/FocusChangeListener.h [hunk 1]
```diff
@@ -34,9 +34,20 @@ static const int FOCUS_NONE = 0;
 static const int FOCUS_GAIN = 1;
 static const int FOCUS_LOSS = -1;
 
+/**
+ * @class 
+ * @brief This class is callback insterface of FocusRequest
```
Comment (chanijjani) [inline framework/include/media/FocusChangeListener.h]: interface..

File: framework/include/media/FocusChangeListener.h [hunk 2]
```diff
@@ -34,9 +34,20 @@ static const int FOCUS_NONE = 0;
...
+ * @details @b #include <media/FocusChangeListener.h>
+ * @since TizenRT v2.0
+ */
 class FocusChangeListener
 {
 public:
+	/**
+	 * @brief Called when focus changed
```
Comment (chanijjani) [inline framework/include/media/FocusChangeListener.h]: when a focus is changed..

File: framework/include/media/FocusChangeListener.h (hunk 1 above)
Comment (junmin-kim) [reply framework/include/media/FocusChangeListener.h]: Fixed it

File: framework/include/media/FocusChangeListener.h (hunk 2 above)
Comment (junmin-kim) [reply framework/include/media/FocusChangeListener.h]: Fixed it

**Comment by seinfra** [comment]
Target : [a49c3d7aa1581b4ae7b3ffd7a84c98d6c548dd42] - Code Rule Check Result:
framework/include/media/FocusManager.h:67: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:VxO)
framework/include/media/FocusManager.h:67: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:OxV)
framework/include/media/FocusManager.h:67: ERROR: [SPC_M_OPR] spaces required around that '<' (ctx:VxV)
framework/include/media/FocusManager.h:67: ERROR: [SPC_M_OPR] spaces required around that '>' (ctx:VxW)
framework/include/media/FocusManager.h:75: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:VxO)
framework/include/media/FocusManager.h:75: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:OxV)
framework/include/media/FocusManager.h:75: ERROR: [SPC_M_OPR] spaces required around that '<' (ctx:VxV)
framework/include/media/FocusManager.h:75: ERROR: [SPC_M_OPR] spaces required around that '>' (ctx:VxW)
framework/include/media/FocusManager.h:81: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:VxO)
framework/include/media/FocusManager.h:81: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:OxV)
framework/include/media/FocusManager.h:81: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:VxO)
framework/include/media/FocusManager.h:81: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:OxV)
framework/include/media/FocusManager.h:81: ERROR: [SPC_M_OPR] spaces required around that '<' (ctx:VxV)
framework/include/media/FocusManager.h:81: ERROR: [SPC_M_OPR] spaces required around that '>' (ctx:VxW)
framework/include/media/FocusManager.h:82: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:VxO)
framework/include/media/FocusManager.h:82: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:OxV)
framework/include/media/FocusManager.h:86: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:OxV)
framework/include/media/FocusManager.h:87: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:OxV)
framework/include/media/FocusManager.h:87: ERROR: [SPC_M_OPR] spaces required around that '<' (ctx:VxV)
framework/include/media/FocusManager.h:87: ERROR: [SPC_M_OPR] spaces required around that '>' (ctx:VxW)
framework/include/media/FocusManager.h:92: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:VxO)
framework/include/media/FocusManager.h:92: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:OxV)
framework/include/media/FocusManager.h:93: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:OxV)
framework/include/media/FocusManager.h:93: ERROR: [SPC_M_OPR] spaces required around that '<' (ctx:VxV)
framework/include/media/FocusManager.h:93: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:VxO)
framework/include/media/FocusManager.h:93: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:OxV)
framework/include/media/FocusManager.h:93: ERROR: [SPC_M_OPR] spaces required around that '<' (ctx:VxV)
framework/include/media/FocusManager.h:93: ERROR: [SPC_M_OPR] need consistent spacing around '>>' (ctx:VxW)
framework/include/media/FocusManager.h:94: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:OxV)
framework/include/media/FocusRequest.h:65: ERROR: [IDT_M_TAB] code indent should use tabs where possible
framework/include/media/FocusRequest.h:65: ERROR: [IDT_M_TAB] please, no space before tabs
framework/include/media/FocusRequest.h:68: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:VxO)
framework/include/media/FocusRequest.h:68: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:OxV)
framework/include/media/FocusRequest.h:68: ERROR: [SPC_M_OPR] spaces required around that '<' (ctx:VxV)
framework/include/media/FocusRequest.h:68: ERROR: [SPC_M_OPR] spaces required around that '>' (ctx:VxW)
framework/include/media/FocusRequest.h:72: ERROR: [IDT_M_TAB] code indent should use tabs where possible
framework/include/media/FocusRequest.h:72: ERROR: [IDT_M_TAB] please, no space before tabs
framework/include/media/FocusRequest.h:75: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:OxV)
framework/include/media/FocusRequest.h:75: ERROR: [SPC_M_OPR] spaces required around that '<' (ctx:VxV)
framework/include/media/FocusRequest.h:75: ERROR: [SPC_M_OPR] spaces required around that '>' (ctx:VxW)
framework/include/media/FocusRequest.h:78: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:OxV)
framework/include/media/FocusRequest.h:79: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:OxV)
framework/include/media/FocusRequest.h:79: ERROR: [SPC_M_OPR] spaces required around that '<' (ctx:VxV)
framework/include/media/FocusRequest.h:79: ERROR: [SPC_M_OPR] spaces required around that '>' (ctx:VxW)
framework/include/media/FocusRequest.h:88: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:OxV)
framework/include/media/FocusRequest.h:95: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:OxV)
framework/include/media/FocusRequest.h:95: ERROR: [SPC_M_OPR] spaces required around that '<' (ctx:VxV)
framework/include/media/FocusRequest.h:95: ERROR: [SPC_M_OPR] spaces required around that '>' (ctx:VxW)
framework/include/media/FocusRequest.h:98: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:OxV)
framework/include/media/FocusRequest.h:99: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:OxV)
framework/include/media/FocusRequest.h:99: ERROR: [SPC_M_OPR] spaces required around that '<' (ctx:VxV)
framework/include/media/FocusRequest.h:99: ERROR: [SPC_M_OPR] spaces required around that '>' (ctx:VxW)

**Comment by seinfra** [comment]
Target : [a49c3d7aa1581b4ae7b3ffd7a84c98d6c548dd42] - Code Rule Check (C++) OK.

**Pull request merged**